lifetimes/estimation.py

```diff
@@ -37,6 +37,117 @@ def _print_params(self):
         return s.strip(', ')
 
 
+class GammaGammaFitter(BaseFitter):
+    def __init__(self, penalizer_coef=0.):
+        self.penalizer_coef = penalizer_coef
+
+    @staticmethod
+    def _negative_log_likelihood(params, frequency, avg_monetary_value, penalizer_coef):
+        if any(i < 0 for i in params):
+            return np.inf
+
+        p, q, v = params
+
+        x = frequency.astype('int')
+        m = avg_monetary_value.astype('int')
+
+        p_0 = np.log(special.gamma(p*x+q)) - np.log(special.gamma(p)) - np.log(special.gamma(x)) - np.log(special.gamma(q))
+        p_1 = q*np.log(v) + (p*x-1)*np.log(m) + p*x*np.log(x) - (p*x+q)*np.log(v + m*x)
```

Comment:
So basically p_1 is this:

<img width="125" alt="screen shot 2015-09-08 at 10 24 00 am" src="https://cloud.githubusercontent.com/assets/166602/9729980/c2636b32-5613-11e5-8155-665b66b07126.png">

if you check wolframalpha for the log equivalent of it http://www.wolframalpha.com/input/?i=log%28+%5B+%28v%5Eq%29+*+%28+m%5E%28p*x-1%29+*+x%5E%28p*x%29+%29+%5D+%2F+%5B+%28v+%2B+m*x%29%5E%28p*x-1%29+%5D+%29+ 

you get

<img width="419" alt="screen shot 2015-09-08 at 10 33 59 am" src="https://cloud.githubusercontent.com/assets/166602/9730200/2912a5d6-5615-11e5-927d-d510b3b358ca.png">

That is equivalent to
```
p_1 = q\*np.log(v) + (p\*x-1)\*np.log(m) + p\*x\*np.log(x) - (p\*x+q)\*np.log(v + m\*x)
```